### ISSUE TITLE
MATMAV setup error

### ISSUE THREAD

**Issue by kumarrt** [mzahana/MATMAV#2]
I am trying to use your MATMAV package for making my pixhawk talk to mocap system. When I run the set up command in matlab, I get the following error. 

Error using mex
gcc: error: ws2_32.lib: No such file or directory

Error in setup (line 24)
    mex -O pnet.c ws2_32.lib -DWIN32

**Comment by mzahana** [comment]
Have you checked that your MATLAB is associated with a compiler?
check the following in matlab command line
```
mex -setup
```

**Comment by kumarrt** [comment]
Yes, I was getting the compiler error initially but I made sure that the compiler is associated with MATLAB. Once I resolved the compiler error, then I started getting the above discussed error.

**Comment by mzahana** [comment]
Sorry. I am not sure what the problem is. Have you tried to Google it?

**Comment by kumarrt** [comment]
I will recheck the compiler once more. Probably, there is an issue with my compiler.

**Comment by paoloelle** [comment]
@kumarrt have you solved the problem? I have the same problem and I did the same steps as yours.
